ecosystem/README.md

```diff
@@ -141,11 +138,16 @@ From there, the following process will happen:
 * Once a SEP has been approved, it goes into FCP which is broadcast to the protocol meeting members
   along with the mailing list.
 
-### FCP -> Active/Final
-* If no major concerns are brought up, the SEP is marked as `Active` or `Final` by your SEP buddy:
-  * If the SEP requires active maintenance, such as having an open schema, it should be marked at
-    `Active`.
-  * Otherwise, the SEP should be marked as `Final`.
+### FCP -> Active
+* If no major concerns are brought up, the SEP is marked as `Active` by your SEP buddy.
+* Ideally there will be a reference implementation exhibiting the behavior and value of the SEP before moving to active state.
+* Active SEPs should be brought into production by ecosystem members.
+* Minor changes may be made as more implementations are brought online highlighting any edge cases.
+
+### Active -> Final
+* Once there are live implementations being used in production, the SEP team can move the SEP to `Final` status.
```

Comment:
Rather than live implementations being the trigger, can we make it a requirement? I think the trigger should be that a decision has been made that the proposal is complete, proven, and won't be extended, and one of the most important attributes to consider when making that decision is if it's being used in live products.